nginx_proxy/Dockerfile

```diff
@@ -1,11 +1,120 @@
 ARG BUILD_FROM
+
+FROM ${BUILD_FROM} as build_modsecurity
+
+ARG NGINX_VERSION=1.16.1
+ARG MODSEC_BRANCH=v3/master
+ARG OWASP_BRANCH=v3.3/master
```

Comment:
Args values are offloaded into `build.json`

Reply:
Thanks for the tip. done.